bosh-stemcell/spec/stemcells/go_agent_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe 'Stemcell with Go Agent', stemcell_image: true do
   describe 'installed by bosh_go_agent' do
-    %w(bosh-agent bosh-agent-rc bosh-blobstore-dav bosh-blobstore-gcs bosh-blobstore-s3).each do |binary|
+    %w(bosh-agent bosh-agent-rc bosh-blobstore-dav bosh-blobstore-gcs bosh-blobstore-s3 bosh-blobstore-az).each do |binary|
```

Comment:
Please adjust the package name to `bosh-blobstore-azure-storage` corresponding to the definition in azure-storage-cli repo.

Reply:
Done.